requirements.yml

```diff
@@ -1,6 +1,8 @@
 ---
 collections:
-  - name: redhat.data_grid
   - name: redhat.rhbk
+  - name: redhat.data_grid
+    version: '>=1.1.4'
+  - name: redhat.sso
```

Comment:
also not needed here

Reply:
Thanks! I didn't see you have removed the sso playbook, removed it now. review once.